test/unit/widgets/buttons.test.ts

```diff
@@ -186,7 +196,58 @@ describe('continueButton', () => {
     expect(continueButton(dom, handler)).toBeTruthy()
   })
 })
+describe('createLinkDiv', () => {
+  const obj = namedNode('https://test.com/#name')
+  const options: RenderAsDivOptions = {
+    deleteFunction: () => {},
+    link: true
+  }
+
+  it('adds a div to the element provided', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children.length).toBeGreaterThan(0)
+  })
+  it('makes the element draggable', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.getAttribute('draggable')).toEqual('true')
+  })
+  it('adds the style....', () => {
+    const options = {}
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children[0].getAttribute('style')).toEqual(style.linkDivStyle)
+  })
+
+  it.skip('adds the deleteFunction of .... deleteButton with Check', () => {
```

Comment:
**issue** this test should not be skipped
You are probably missing `deleteButton.setAttribute('role', 'button')` to make this test work